sys-boot/plymouth/plymouth-24.004.60.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+
+inherit meson readme.gentoo-r1 flag-o-matic
+
+SRC_URI="https://dev.gentoo.org/~aidecoe/distfiles/${CATEGORY}/${PN}/gentoo-logo.png"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/plymouth/plymouth"
+else
+	SRC_URI="${SRC_URI} https://www.freedesktop.org/software/plymouth/releases/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+fi
+
+DESCRIPTION="Graphical boot animation (splash) and logger"
+HOMEPAGE="https://cgit.freedesktop.org/plymouth/"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="debug +drm +gtk +pango selinux freetype +split-usr +udev doc upstart-monitoring systemd"
+
+BDEPEND="
+	app-text/docbook-xsl-stylesheets
+	>=dev-build/meson-0.62
+	dev-libs/libxslt
+	virtual/pkgconfig
+"
+
+DEPEND="
+	dev-libs/libevdev
+	drm? ( x11-libs/libdrm )
+	elibc_musl? ( sys-libs/rpmatch-standalone )
+	freetype? ( media-libs/freetype:2 )
+	gtk? (
+		dev-libs/glib:2
+		x11-libs/cairo
+		>=x11-libs/gtk+-3.14:3[X]
+	)
+	>=media-libs/libpng-1.2.16:=
+	pango? (
+		x11-libs/cairo
+		>=x11-libs/pango-1.21
+	)
+	systemd? ( sys-apps/systemd )
+	udev? ( virtual/libudev )
+	upstart-monitoring? (
+		sys-apps/dbus
+		sys-libs/ncurses
+	)
+	virtual/libc
+	x11-libs/libxkbcommon
+	x11-misc/xkeyboard-config
+"
+
+RDEPEND="${DEPEND}
+	selinux? ( sec-policy/selinux-plymouthd )
+	>sys-kernel/dracut-0.37-r3
+	udev? ( virtual/udev )
+"
```

Comment:
Unconditional dependencies and conditional dependencies should be grouped together. Sorry for not being clear here.
``` suggestion
DEPEND="
	dev-libs/libevdev
	>=media-libs/libpng-1.2.16:=
	virtual/libc
	x11-libs/libxkbcommon
	x11-misc/xkeyboard-config
	drm? ( x11-libs/libdrm )
	elibc_musl? ( sys-libs/rpmatch-standalone )
	freetype? ( media-libs/freetype:2 )
	gtk? (
		dev-libs/glib:2
		x11-libs/cairo
		>=x11-libs/gtk+-3.14:3[X]
	)
	pango? (
		x11-libs/cairo
		>=x11-libs/pango-1.21
	)
	systemd? ( sys-apps/systemd )
	udev? ( virtual/libudev )
	upstart-monitoring? (
		sys-apps/dbus
		sys-libs/ncurses
	)
"

RDEPEND="${DEPEND}
	>sys-kernel/dracut-0.37-r3
	selinux? ( sec-policy/selinux-plymouthd )
	udev? ( virtual/udev )
"
```